Eval_expr.c

```diff
@@ -0,0 +1,92 @@
+#define MAX 100
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+
+
+int Op_eval(int a, int b, char op) {
+    switch (op)
+    {
+    case '+': return a+b;
+        break;
+    case '-': return a-b;
+        break;
+    case '*': return a*b;
+        break;
+    case '/': if (b == 0) {
+            printf("Error: Division by zero.\n");
+            exit(1);
+        }
+        return a / b;
+        break;
+    default: return 0;
+        
+    }
+}
+
+int op_Check(char ch) {
+    return ch == '+' || ch == '-' || ch == '*' || ch == '/';
+}
+
+int Exp_sol(char* expr) {
+    int val[MAX], tp_val = -1;
+    char ops[MAX], tp_op = -1;
+
+    for (int i = 0; expr[i] != '\0'; i++) {
+        if (isspace(expr[i])) continue;

```

Comment:
Please implement functions like isspace as it would be helpful for learning and understanding how these work behind the scene